src/components/collection_voting_slab/ResearchProposalVoteSlab.js

```diff
@@ -14,6 +14,9 @@
 import VotesPieChart from '../common/VotesPieChart';
 import VoteSummaryTable from '../vote_summary_table/VoteSummaryTable';
 import CollectionAlgorithmDecision from '../CollectionAlgorithmDecision';
+import {convertLabelToKey} from "../../libs/utils";
+import {ScrollToTopButton} from "../ScrollButton";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9655076)
```suggestion
import {ScrollToTopButton} from '../ScrollButton';
```